modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/challenge/questions/model/ServerChallengeModel.java

```diff
@@ -0,0 +1,95 @@
+/*
+ *Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *WSO2 Inc. licenses this file to you under the Apache License,
+ *Version 2.0 (the "License"); you may not use this file except
+ *in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing,
+ *software distributed under the License is distributed on an
+ *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *KIND, either express or implied.  See the License for the
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+
+package org.wso2.identity.integration.test.rest.api.challenge.questions.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+/**
+ * This class is used to store data about the challenge questions.
+ * Class will be sent in the response.
+ * This is mapped to the request body.
+ */
+public class ServerChallengeModel {
+    @Expose
+    @SerializedName("questionSetId")
+    private String questionSetId;
+    @Expose
+    @SerializedName("questions")
+    private List<Questions> questions;
+
+    @Override
+    public String toString() {
+        return "{" +
```

Comment:
Add an empty line at the beginning of the method. Check for other places as well.